src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -90,10 +93,42 @@ const StateModel = Backbone.Model.extend({
     }));
   },
   isSelected(model) {
-    const list = this.isFlowType() ? this.get('selectedFlows') : this.get('selectedActions');
+    const list = this.getSelectedList();
 
     return list[model.id];
   },
+  getSelected(collection) {
+    if (!collection) return;
+
+    const list = this.getSelectedList();
+    const collectionSelected = _.reduce(_.keys(list), (selected, item) => {
+      if (list[item] && collection.get(item)) {
+        return selected.concat([item]);
```

Comment:
`selected.push(collection.get(item))` I think.. does the entity work if it's just a collection of id strings?  I assumed it'd either need to be models or `{ id: 'idstring' }`

Reply:
Ah thanks. I hadn't dug into the model attributes to verify it was building the collection correctly.